.github/workflows/ci.yml

```diff
@@ -2,17 +2,17 @@ name: CI
 
 on:
   push:
-    branches:
-      - main
-    paths:
-      - src/**
-  pull_request:
-    paths:
-      - src/**
+  #   branches:
+  #     - main
```

Comment:
Possible to do branches all instead of main?

Reply:
Ah, good catch! Fixed